device/src/esp32_run.rs

```diff
@@ -159,10 +143,7 @@ where
                 if downstream_serial.find_and_remove_magic_bytes() {
                     downstream_active = true;
                     ui.set_downstream_connection_state(true);
-                    sends_upstream.push(DeviceSendMessage::Debug {
-                        message: "Device read magic bytes from another device!".to_string(),
-                        device: frost_signer.clone().device_id(),
-                    });
+                    sends_upstream.debug("Device read magic bytes from another device!");
```

Comment:
Could we maybe change .`debug` to `.send_debug`?

Reply:
I don't mind either way so I did it.